src/test-harness/SparkEthereumTests.sol

```diff
@@ -89,8 +94,8 @@ abstract contract SparkEthereumTests is SparklendTests {
             ChainId chainId = ChainIdUtils.fromDomain(chainSpellMetadata[allChains[i]].domain);
             if (chainId == ChainIdUtils.Ethereum()) continue;  // Checking only foreign payloads
             address payload = chainSpellMetadata[chainId].payload;
-            if (payload != address(0)) {
-                // A payload is defined for this domain
+            if (payload != address(0) && !chainSpellMetadata[chainId].notPreDeployed) {
```

Comment:
Why was this added? We want this test to be failing until the payload is deployed.

Reply:
Removed